library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/recorder/ScreenRecorderTest.kt

```diff
@@ -93,6 +95,29 @@ internal class ScreenRecorderTest {
             .isEqualTo(mockDefaultCallback)
     }
 
+    @Test
+    fun `M register the RecorderWindowCallback W startRecording{default callback is null}`(
+        forge: Forge
+    ) {
+        // Given
+        val mockWindow: Window = mock()
+        val mockActivity = mockActivity(
+            forge,
+            window = mockWindow,
+            defaultWindowCallback = null
+        )
+
+        // When
+        testedRecorder.startRecording(mockActivity)
+
+        // Then
+        val captor = argumentCaptor<Window.Callback>()
```

Comment:
minor: we can use lambda syntax with `captor == this`

```
argumentCaptor<Window.Callback>() {
   ...
}
```

Reply:
I find it a bit unreadable for me that way :). That's the reason I prefer to non-lambda way.